tests/test_api_v1.py

```diff
@@ -1512,16 +1512,6 @@ def test_fetch_request_content_manifest_non_implemented_type(
         p["type"] = pkg_type
 
     image_contents = []
-    if pkg_type == "gomod":
-        # gomod is a special case where pre-icm requests will have ICMs with
```

Comment:
Note that this is no longer true. We can not rely on the existence of a package with the same name as the module. This could be wrong since the toplevel purl is for a package.